src/buttonBlock.js

```diff
@@ -0,0 +1,41 @@
+export default class ButtonBlock {
+  editor;
```

Comment:
the two class properties: editor and opts are not needed and can be removed for a cleaner code.

Reply:
@adiux you can check now.